### PR TITLE
Feature/network fixes

### PR DESCRIPTION
**Pull request by tacozMacleo**
Have Changed the Network flow, to be more thread safe. (The Rest require a redesign)
And added a poke, for the send Thread, so it known something have changed, and it might be able to send again.

### CODE REVIEW

**Comment by tacozMacleo** [comment]
And It seams that development branch did not have the feature/cleanup merged ind. So the seluxit_rpc have been downgraded from a class to a utils lib. Change some structure to make it easier readable, and some redundant code that was not used, and many more.

File: wappsto/connection/receive_data.py [hunk 1]
```diff
@@ -11,7 +11,8 @@
 from . import message_data
 from json.decoder import JSONDecodeError
 
-RECEIVE_SIZE = 1024
+# RECEIVE_SIZE = 1024
```
Comment (AndreasBomholtz) [inline wappsto/connection/receive_data.py:14]: Remove old code.

File: wappsto/modules/value.py [hunk 2]
```diff
@@ -413,7 +418,8 @@ def update(self, data_value, timestamp=get_now()):
             state_id=state.uuid,
             verb=message_data.PUT
         )
-        self.parent.parent.conn.send_data.send_report(msg)
+        # self.parent.parent.conn.send_data.send_report(msg)
```
Comment (AndreasBomholtz) [inline wappsto/modules/value.py:421]: Delete old code.

File: wappsto/__init__.py [hunk 3]
```diff
@@ -217,19 +224,20 @@ def start(self, address="wappsto.com", port=11006, automatic_trace=False, blocki
 
         Args:
             address: Address to connect the service to.
-                (default: {"wappsto.com"})
-            port: Port to connect the address to. (default: {11006})
+                     (default: "wappsto.com")
```
Comment (AndreasBomholtz) [inline wappsto/__init__.py:227]: Default address is collector.wappsto.com

File: wappsto/__init__.py [hunk 4]
```diff
@@ -217,19 +224,20 @@ def start(self, address="wappsto.com", port=11006, automatic_trace=False, blocki
...
+            port: Port to connect the address to.
+                  (default: 11006)
```
Comment (AndreasBomholtz) [inline wappsto/__init__.py:229]: The default port is now 443

**Review by AndreasBomholtz — CHANGES_REQUESTED** [review]

**Pull request closed without merging**